testing/go/create_function_test.go

```diff
@@ -650,5 +650,62 @@ $$ LANGUAGE plpgsql;`},
 				},
 			},
 		},
+		{
+			Name: "Branching",
```

Comment:
Do we have tests that merging a function created on another branch brings it over?

Reply:
We do now!